internal/slicer/report_test.go

```diff
@@ -48,6 +48,12 @@ var sampleLink = fsutil.Entry{
 	Link: "/base/exampleFile",
 }
 
+var sampleFileMutated = fsutil.Entry{
+	Path: "/base/exampleFile",
+	Hash: "exampleFile_changed_hash",
```

Comment:
Can we define this in relation to the other test? That would make it impossible for someone in the future to accidentally change the size in `sampleFile` and make the test ineffective. So,
```go
Path: sampleFile.Path
Hash: sampleFile.Hash + "_changed"
Size: sampleFile.Size + 10
```